application/comit_node/tests/rfc003_bitcoin_htlc.rs

```diff
@@ -98,23 +107,23 @@ fn redeem_htlc_with_secret() {
 
     let container = docker.run(BitcoinCore::default());
     let client = tc_bitcoincore_client::new(&container);
-    client.generate(432).unwrap().unwrap();
+    client.generate(432, None).unwrap();
```

Comment:
❓Do you remember why we generate 432 blocks? Maybe we can get a performance boost if we only generate 1?
```suggestion
    client.generate(1, None).unwrap();
```

Reply:
I think that was because of the misconception of enabling segwit. I'd propose cleanup with https://github.com/coblox/blockchain-contracts/issues/3.